src/lib/versionRequirements.ts

```diff
@@ -0,0 +1,7 @@
+const VERSIONS = {
+	EXTRA_SUPPORT: {
```

Comment:
This name isn't really descriptive enough for me. Looking at this file I (and without context) I don't really have any idea what this means, other than *something* to do with the jetson at v2.47

Reply:
I've removed this file and instead added a single value to constants.ts

https://github.com/balena-io/balena-supervisor/blob/cbe4fc4e5b276e9965402f105069d43ab2d3fc11/src/lib/constants.ts#L68-L69